src/main/java/io/fabric8/maven/docker/model/ContainerDetails.java

```diff
@@ -1,17 +1,12 @@
 package io.fabric8.maven.docker.model;
 
-import java.time.Instant;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.Set;
-
 import com.google.common.base.Joiner;
-import com.google.common.base.Preconditions;
 import com.google.gson.JsonArray;
 import com.google.gson.JsonObject;
 
+import java.time.Instant;
+import java.util.*;
```

Comment:
Could you please revert these import changes?

Please [configure your IDE to not add wildcard imports](https://stackoverflow.com/a/25916603/2830116)